docs/localization.rst

```diff
@@ -71,17 +96,30 @@ code of the language you are adding.
    https://product-details.mozilla.org/1.0/languages.json
    and place it at ``kuma/settings/languages.json``.
 
-#. Add the locale to the ``locale/`` folder::
+#. Add the locale to the ``locale/`` folder:
+
+    **In Docker**::
```

Comment:
There is one too many leading spaces, turning this from a command reference into a quoted command reference.  Remove the extra leading space.